### ISSUE TITLE
php artisan opcache:optimize not working

### ISSUE THREAD

**Issue by gregmsanderson** [appstract/laravel-opcache#82]
I've just been trying this to manage the opcache. As far as I can tell, opcache is working correctly (it appears in the phpinfo() as enabled, the site is indeed quicker with it enabled, and using the artisan command to get its status also confirms it is running and data is being cached when requested page-by-page)

That is filled from requesting pages normally in the browser. So it does work, but it means the first request is really slow after each change.

So I tried your 'php artisan opcache:optimize' command to _pre_-generate the cache before the first visit. But it reports 0 files as being optimized. Does that mean 0 are being _cached_? That doesn't _sound_ right, but I may have misunderstood what it is trying to do or whether caching is separate to optimizing.

For example:

C:\apps\example.com>php artisan opcache:optimize
Optimize started, this can take a while...
0 of 930 files optimized

... and showing the stats command first to show it was working ...
```
...
Statistics:
+----------------------+---------------------+
| option               | value               |
+----------------------+---------------------+
| num_cached_scripts   | 1102                |
| num_cached_keys      | 2392                |
| max_cached_keys      | 130987              |
| hits                 | 15865               |
| start_time           | 2019-02-25 19:59:28 |
| last_restart_time    | 2019-02-25 20:12:48 |
| oom_restarts         | 0                   |
| hash_restarts        | 0                   |
| manual_restarts      | 2                   |
| misses               | 1102                |
| blacklist_misses     | 0                   |
| blacklist_miss_ratio | 0                   |
| opcache_hit_rate     | 93.505039193729     |
+----------------------+---------------------+

C:\apps\example.com>php artisan opcache:optimize
Optimize started, this can take a while...
0 of 930 files optimized
```

Hmm.

So I tried extracting the URL it seems to make, and requested that in the browser rather than the terminal. So the one like:
example.com/opcache-api/optimize?key=eybiglongkeyhere

That returns:

```
{
"result": {
"total_files_count": 930,
"compiled_count": 0
}
}
```

So the same result, but here is says 'compiled' rather than 'optimized'. Presumably that means the same.

The log does not contain any errors so it's not clear if it is trying to access routes and being blocked or quite what it is trying to do internally. Or perhaps it _is_ working correctly, and 0 is not a problem (the misunderstood situation)?

Thanks!

**Comment by stexine** [comment]
I have excatly the same problem after upgrading to laravel 5.8.

**Comment by ovanschie** [comment]
Should be fixed in 3.0
